.travis.yml

```diff
@@ -26,11 +26,20 @@ matrix:
 
       # install database
       addons:
-        postgresql: "9.5"
+        postgresql: "12"
+        apt:
+          packages:
+          - postgresql-12
+          - postgresql-client-12
+      env:
+        global:
+        - PGVER=12
+        - PGPORT=5433
```

Comment:
After you rebase remaster into this branch, could you please add `PG_HOST` and `PG_PORT` in here and remove them from the Travis UI (https://travis-ci.com/github/bravetechnologycoop/BraveButtons/settings)? Thanks!!!